src/plugins/query_enhancements/server/connections/managers/prometheus_manager.ts

```diff
@@ -97,6 +104,8 @@ class PrometheusManager extends BaseConnectionManager<OpenSearchClient> {
         return `${BASE_RESOURCE_API}/alerts`;
       case PROMETHEUS_RESOURCE_TYPES.LABEL_VALUES:
         return `${BASE_RESOURCE_API}/label/${resourceName}/values`;
+      case PROMETHEUS_RESOURCE_TYPES.METRICS:
+        return `${BASE_RESOURCE_API}/label/__name__/values`;
```

Comment:
no strong opinion but can it reuse the `PROMETHEUS_RESOURCE_TYPES.LABEL_VALUES`? since on prometheus side it's the same API

Reply:
I wanted to abstract that implementation detail for our consumers.